translations/base-fr.yaml

```diff
@@ -506,49 +541,75 @@ buildings:
         default:
             name: &painter Peintre
             description: &painter_desc Colorie entièrement la forme de gauche avec la couleur de droite.
+
+        mirrored:
+            name: *painter
+            description: Colorie entièrement la forme de gauche avec la couleur de gauche.
+
         double:
             name: Peintre (double)
             description: Colorie les deux formes de gauche avec la couleur de droite.
+
         quad:
             name: Peintre (quadruple)
             description: Colorie chaque quadrant d’une forme avec une couleur différente.
-        mirrored:
-            name: *painter
-            description: *painter_desc
 
     trash:
         default:
             name: &trash Poubelle
             description: Accepte des formes de n’importe quel côté et les détruit… pour toujours.
 
-        storage:
+    storage:
+        default:
             name: Stockage
             description: Stocke les formes en trop jusqu’à une certaine capacité. Peut être utilisé pour absorber un surplus.
-    hub:
-        deliver: Livrez
-        toUnlock: pour débloquer
-        levelShortcut: NV
+
     wire:
         default:
             name: Câble
-            description: Permet de transporter de l’énergie.
-    advanced_processor:
+            description: &wire_desc Transfère des signaux, qui peuvent être des éléments, des couleurs ou des booléens (1 / 0). Les câbles de couleurs différentes ne se connectent pas.
+
+        second:
+            name: *wire
+            description: *wire_desc
+
+    wire_tunnel:
         default:
-            name: Inverseur de couleur
-            description: Accepte une couleur ou une forme, et l’inverse.
-    energy_generator:
-        deliver: Livrer
-        toGenerateEnergy: Pour
+            name: &wire_tunnel Tunnel à câble
+            description: Permet de croiser deux câbles sans les connecter entre eux.
+
+    constant_signal:
         default:
-            name: Générateur d’énergie
-            description: Génère de l’énergie en consommant des formes.
-    wire_crossings:
+            name: &constant_signal Constante
+            description: Émets un signal constant, qui peut être une forme, une couleur ou un booléen (1 / 0).
+
+    lever:
         default:
-            name: Duplicateur de câble
-            description: Sépare un câble en deux.
-        merger:
-            name: Fusionneur de câble
-            description: Fusionne deux câbles en un seul.
+            name: &lever Interrupteur
+            description: Peut être basculé pour émettre un signal booléen (1 / 0) dans le calque de câblage, qui peut être utilisé pour contrôler un filtre de forme par exemple.
+
+    logic_gate:
+        default:
+            name: Porte ET
+            description: Émets un booléen "1" si les deux entrées sont vraies.
```

Comment:
Could you use curly quotes (throughout)? I know they're horrid but that would be consistent with the rest of the file. (Or, like, guillemets, but aaaarghhh.)